src/@next/components/molecules/ColorSelection/ColorSelection.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { IProps, IColor, LocalStorageKeys, IPosition } from "./types";
+import { ColorItem } from "./ColorItem";
+import "./index.scss";
+
+const getDataFromStorage = () =>
+  JSON.parse(localStorage.getItem(LocalStorageKeys.CabinetColor)) || {};
+
+export const ColorSelection: React.FC<IProps> = ({
+  colors,
+  position,
+  collectionId,
+}) => {
+  const [selectedColor, setSelectedColor] = useState(getDataFromStorage());
+
+  const onChangeColor = (type: IPosition, colorID: number) => {
+    const color = {
+      ...selectedColor,
+      [type]: colorID,
+      collectionId: collectionId,
+    };
+    localStorage.setItem(LocalStorageKeys.CabinetColor, JSON.stringify(color));
+    setSelectedColor(color);
+  };
```

Comment:
I left a comment on the main page about how we can store this to be more flexible.

Reply:
Done